install_from_src

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+
+clean_slate()
+{
+	echo "                      =============================================="
+	echo "                      If you are within a conda env please deactivate it and come back"
```

Comment:
If we move all the commands in this script to README we can just explain how customers can install and run etc.